sky/templates/kubernetes-ray.yml.j2

```diff
@@ -417,7 +426,18 @@ setup_commands:
   # Line 'sudo grep ..': set the number of threads per process to unlimited to avoid ray job submit stucking issue when the number of running ray jobs increase.
   # Line 'mkdir -p ..': disable host key check
   # Line 'python3 -c ..': patch the buggy ray files and enable `-o allow_other` option for `goofys`
-  - sudo DEBIAN_FRONTEND=noninteractive apt install lsof gcc patch pciutils rsync fuse curl -y;
+  - |
+    PACKAGES="lsof gcc patch pciutils rsync fuse curl";
```

Comment:
nit: is `lsof` still needed if we are going to remove it?

Reply:
Good point, removed now.